pkg/minikube/cruntime/cruntime.go

```diff
@@ -225,9 +225,10 @@ func disableOthers(me Manager, cr CommandRunner) error {
 			klog.Infof("skipping containerd shutdown because we are bound to it")
 			continue
 		}
-
-		// runtime is already disabled, nothing to do.
-		if !r.Active() {
+		// in case of docker, if other runtime are already not active we are sure it is disabled, nothing to do.
+		// because #11515 for non-docker runtimes, we gotta ensure Docker is disabled and can not just check if it is not active
+		// since it is enabled by default in the current base image and it keeps coming back to life
+		if me.Name() == "Docker" && !r.Active() {
```

Comment:
this issue is about `containerd` runtime. Isn't `me` in this case `containerd` and this line always false?

Reply:
this looks confusing but , 

the if statement says if runtime is "Docker" and the other things are Not Active "Continue" (Skip the for loop)
which means for Containerd and Crio do not skip Disabling Others. and do a Hard Disable later in the code

Reply:
got it. yeah, it's confusing.